Classes/ViewHelpers/ParseMetadataWrapViewHelper.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * (c) Kitodo. Key to digital objects e.V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo and TYPO3 projects.
+ *
+ * @license GNU General Public License version 3 or later.
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ */
+
+namespace Kitodo\Dlf\ViewHelpers;
+
+use TYPO3Fluid\Fluid\Core\Rendering\RenderingContextInterface;
+use TYPO3Fluid\Fluid\Core\ViewHelper\AbstractViewHelper;
+use TYPO3\CMS\Core\TypoScript\Parser\TypoScriptParser;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Extbase\Configuration\ConfigurationManager;
+
+class ParseMetadataWrapViewHelper extends AbstractViewHelper
+{
+    protected $escapeOutput = false;
+
+    public static function renderStatic(array $arguments, \Closure $renderChildrenClosure, RenderingContextInterface $renderingContext)
+    {
+        $parser = GeneralUtility::makeInstance(TypoScriptParser::class);
+        $parser->parse($renderChildrenClosure());
+        return [
```

Comment:
Not sure how to fix this return value. Usually it's string. Please think again about it.

Reply:
Oh I see. Please have a look at my fix, where I combine TypoScript parsing and variable creation into a single viewhelper.